tests/FockSpace/FockSpace_test.cpp

```diff
@@ -466,3 +529,41 @@ BOOST_AUTO_TEST_CASE ( FockSpace_EvaluateOperator_diagonal_vs_no_diagonal) {
     // Test if non-diagonal evaluation and diagonal evaluations are correct
     BOOST_CHECK(hamiltonian.isApprox(hamiltonian_no_diagonal + GQCP::SquareMatrix<double>(hamiltonian_diagonal.asDiagonal())));
 }
+
+
+
+/**
+ *  Perform a matrix vector product evaluation of a one-, two-electron operator and the Hamiltonian in the Fock space
+ *  and compare these to the matrix vector product of the actual dense evaluations.
```

Comment:
The documentation can be on one line

```suggestion
 *  and compare these to the matrix-vector product of the actual dense evaluations
```